lib/json_api_client/query/builder.rb

```diff
@@ -7,13 +7,14 @@ class Builder
       def initialize(klass)
         @klass = klass
         @pagination_params = {}
+        @filters = {}
         @base_params = {}
         @includes = []
         @orders = []
       end
 
       def where(conditions = {})
-        @base_params.merge!(conditions)
+        @filters.merge!(conditions)
```

Comment:
I don't think so - they all get merged in `params`.  But it does seem that @base_params isn't doing anything anymore.